modules/ROOT/pages/build-app-practices.adoc

```diff
@@ -77,13 +77,13 @@ If you use Anypoint Connector for File (File Connector) and share part of the pa
 ----
 
 === HTTP and HTTPS
-For HTTP and HTTPS, use the following placeholders, depending on the deployment model:
 
-* `${http.port}` +
-When deploying to CloudHub or Hybrid using HTTP (port=8081) or HTTPS (port=8082)
+HTTP and HTTPS property placeholders are reserved in CloudHub deployments. Use of the same placeholders is recommended for on-premises deployments.
 
-* `${http.private.port}` +
-When deploying to CloudHub using HTTP (port=8091) or HTTPS (port=8092) and using a Dedicated Load Balancer
+* In CloudHub, `${http.port}` maps to the setting `port=8081`, and `${https.port}` maps to `port=8082`.
```

Comment:
We should mention that those ports are the default used by the shared load balancer in CH. Not sure if there is any doc in CH that explains that

Reply:
```suggestion
* In CloudHub, `${http.port}` maps to the setting `port=8081`, and `${https.port}` maps to `port=8082`. The ports are the defaults used by the shared load balancer in CloudHub. 
```

Reply:
Revised the initial paragraph to include info on the load balancers. There's a link to the CH info in "For details, ..." at the end of this section.

Reply:
@facunlk can you check the changes and approve if warranted. Same changes are also in https://github.com/mulesoft/docs-mule-runtime/pull/2120/files. Thx!